tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -211,60 +213,62 @@ that covers 800 nm-805 nm might have a FWHM of 2.5 and a wavelength value of 802
 
 The HDF5 dataset that we are working with in this activity contains more 
 information than we need to work with. For example, we don't necessarily need 
-to process all 426 bands - if we are interested in creating a product like NDVI
-which only users bands in the near infra-red and red portions of the spectrum. 
-Or we might only be interested in a spatial subset of the data - perhaps a region 
-where we have plots in the field.
+to process all 426 bands available in a full NEON hyperspectral reflectance file 
+- if we are interested in creating a product like NDVI which only users bands in 
+the near infra-red and red portions of the spectrum. Or we might only be interested 
+in a spatial subset of the data - perhaps a region where we have plots in the field.
 
 The HDF5 format allows us to slice (or subset) the data - quickly extracting the 
 subset that we need to process. Let's extract one of the green bands in our 
-dataset - band 34. By the way - what is the center wavelength value associated 
-with band 34? hint `wavelengths[34]`. How do we know this band is a green band 
+dataset - band 9. By the way - what is the center wavelength value associated 
+with band 9? hint: `wavelengths[9]`. How do we know this band is a green band 
```

Comment:
Following comment above about hints and giving people time to think.  I'd start a new paragraph just before, "By the way" and then with the Hint.  And capitalize Hint. And again with the next question.

Reply:
Good idea. I have done this here and will add this to the tutorial update checklist.